src/installer/Directory.Build.props

```diff
@@ -51,19 +51,9 @@
     <TargetArchitecture  Condition="'$(TargetArchitecture)' == '' AND '$(BuildArchitecture)' == 'arm64'">$(BuildArchitecture)</TargetArchitecture>
     <TargetArchitecture  Condition="'$(TargetArchitecture)' == ''">x64</TargetArchitecture>
     <Platform Condition="'$(Platform)'==''">$(TargetArchitecture)</Platform>
+    <TargetOS Condition="'$(TargetOS)' == 'Linux'">Unix</TargetOS>
```

Comment:
Does this actually work? When passing in TargetOS, it's a global property which can't be overwritten.